File: app/src/main/java/com/ilyap/yuta/models/ProjectDto.java
package com.ilyap.yuta.models;

import lombok.Value;

@Value
public class ProjectDto {
    int id;
    String photoUrl;
    String name;
    String technicalTaskUrl;
    String status;
    String stringDeadline;
    String description;
    User manager;
    Team team;
}

File: app/src/main/java/com/ilyap/yuta/domain/model/ProjectDto.kt
package com.ilyap.yuta.domain.model

data class ProjectDto(
    val id: Int,
    val photo: String?,
    val name: String,
    val technicalTask: String?,
    val status: String,
    val stringDeadline: String,
    val description: String,
    val manager: User,
    val team: Team?
)
